torch/csrc/jit/serialization/export_bytecode.h

```diff
@@ -16,9 +20,11 @@ struct ExportedFunction {
   ExportedFunction(
       const Module& m,
       const Function& f,
-      std::unique_ptr<Graph> g,
+      std::unique_ptr<Graph>&& g,
```

Comment:
We don't conditionally move the graph here, because we always move it. Usually `&&` express the intension that an argument maybe moved or not.